app.js

```diff
@@ -31,15 +30,27 @@ const rewriteUnsupportedBrowserMethods = (req, res, next) => {
 
 app.use('/public', staticDir);
 app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
 app.use(cookieParser());
 app.use(express.static('public'));
 
+app.use(
+	session({
+		name: 'AuthCookie',
+		secret: process.env.authSecret,
+		saveUninitialized: false,
+		resave: false,
+		cookie: { maxAge: 6000000 }
+	})
+);
+
```

Comment:
I am falling back to express session, since I don't to risk points off because I have used JWT. I have asked professor about JWT and he said it is fine to use it but I don't to play with our project since this is not my personal project

Reply:
I  have tested it rigourously but couldn't find any auth bugs but PLEASE MAKE SURE TO TEST EVERYTHING RIGHT NOW,  so that moving forward we will not have any problems with this.